### PR TITLE
fix sqlite version to avoid an installation that have been happening …

### PR DESCRIPTION
**Pull request by settinghead**
…on some students' vagrant machines

### CODE REVIEW

**Review by vaz — CHANGES_REQUESTED** [review]
I don't think there's any reason to keep sqlite3 in here at all. Suggesting we just remove it. Related: lighthouse-labs/2016-web-curriculum-activities#763

**Comment by vaz** [comment]
Related: lighthouse-labs/2016-web-curriculum-activities#763

**Comment by Harrisandwich** [comment]
I'm going merge this later this week, because it's small and will at least keep things going smoothly until we go through with removing the dependancy. If anyone objects (@vaz) speak now/before wednesday

File: package.json [hunk 1]
```diff
@@ -24,6 +24,6 @@
   },
   "devDependencies": {
     "nodemon": "^1.9.2",
-    "sqlite3": "^3.1.4"
+    "sqlite3": "3.1.4"
```
Comment (daegren) [inline package.json:27]: Just remove this instead, as per @vaz's comment

**Review by daegren — CHANGES_REQUESTED** [review]

**Pull request closed without merging**